group_vars/matrix_servers

```diff
@@ -2557,6 +2557,38 @@ matrix_ma1sd_database_password: "{{ '%s' | format(matrix_homeserver_generic_secr
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-media-repo
+#
+######################################################################
+
+matrix_media_repo_enabled: false
+matrix_media_repo_identifier: matrix-media-repo
```

Comment:
It may be better if this variable is moved to `defaults/main.yml`.

For other roles, that's where we usually store it, with some sane default value.

We also usually make roles use the identifier variable for:

- configuring the container name (`--name={{ matrix_media_repo_identifier }}`) in the systemd service
- configuring the base path that the role uses by default. `group_vars/*` usually overwrite this to something more "playbook-specific". Most of our Matrix roles directly use `matrix_base_path` though - there's no need for such stuff, I suppose.

-----

Generally, all `matrix_media_repo_*` variables are expected to have some default in the role itself. Looks like this one is an exception (only existing in group vars), which is somewhat confusing.

Reply:
Huh, for some reason I could not find any of the matrix roles that has their identifiers declared in `defaults/main.yml` as you described. I did see that pattern for some of the galaxy devture roles however.

I agree though that its probably a good idea to have `matrix_media_repo_identifier` declared in `defaults/main.yml`. I'll follow the current pattern of re-declaring `matrix_media_repo_identifier` in `group_vars/matrix_servers` as its done for all other roles. If you prefer that  `matrix_media_repo_identifier` is only declared in `defaults/main.yml`, let me know and I will make further changes.

Reply:
I think having a default identifier of `matrix-media-repo` and not redeclaring (to the same thing?) it is probably okay too.